packages/repository-tests/src/crud-test-suite.ts

```diff
@@ -33,6 +33,7 @@ export function crudRepositoryTestSuite(
     freeFormProperties: true,
     emptyValue: undefined,
     supportsTransactions: true,
+    supportInclusionResolvers: true,
```

Comment:
```suggestion
    supportsInclusionResolvers: true,
```

For consistency with `features.supportsTransactions`.